rust/rope/src/rope.rs

```diff
@@ -730,6 +730,16 @@ impl<'a> From<&'a Rope> for String {
     }
 }
 
+impl fmt::Display for Rope {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let mut ci = self.iter_chunks_all();
+        for s in ci.next() {
+            write!(f, "{}", s).is_ok();
```

Comment:
You should branch on `f.alternate()` like the impl of `fmt::Debug` and the other pull request.

Reply:
In my opinion an alternate representation isn't strictly necessary, that's more a fancy bonus feature.

One issue here though is that we want to propagate any error that occur in the `write!` macro; the `is_ok()` call basically just ignores them, we should be able to just do `write!(...)?;` and the error will be handled correctly.

Reply:
I read in @cmyr comment somewhere it isn't super important. Hence I ignored it.